### ISSUE TITLE
Nemo does not remember password to reconnect to local NAS, despite being asked to

### ISSUE THREAD

**Issue by kauffy** [linuxmint/nemo#1864]
```
 * Nemo 3.6.5
 * Windowed Nemo
 * Mint 18.3
 * nVidia GTX960 - Proprietary Driver
 * 64 bit
 ```

**Issue**
Nemo does not remember password to reconnect to local NAS, despite being asked to

**Steps to reproduce**
Open Nemo.
Click Network on the left-hand side, under Network.
(On my system, two devices are visible-- this machine, and the NAS)
Double-click the NAS.
"Enter your name and password for the server "NameOfNAS""
Select Registered User.
Enter Correct Credentials for Username and Password.
Select Remember forever.
Click Connect.
NAS will open, and can be accessed normally.
Reboot your system (this will certainly cause it), or just wait a while (unsure if minutes, hours, days).
--- Time Elapsed or Reboot ---
Click Network.
Double-click the NAS.
Prompted again for Username and Password.

**Expected behaviour**
Selecting "Remember password forever" would remember password forever.

**Other information**

**Comment by midwire** [comment]
I'm having the same problem. Anyone?

**Comment by leigh123linux** [comment]
Gnome-keyring does the remember password not nemo.

**Comment by OzzieIsaacs** [comment]
I created a new issue for this, as I'm also having this problem, but only after restart (#2640)
